chore(docs): clean up and add new content (#15003)
immich-app/immich: docs/docs/partials/_storage-template.md

If you want to change the storage template during the initial setup, first enable the feature.

<img src={require('./img/enable-storage-template.png').default} width="80%" title="Enable Storage Template Setting" />

Then, customize your storage template.

> 80

Reviewer: 80?

<img src={require('./img/storage-template.png').default} width="80%" title="Storage Template Setting" />

:::info

> `Storage Template Migration` job can be run after enabling the feature or changing the content of template, in order to apply the change to the existing library.

Reviewer: ```suggestion
The `Storage Template Migration` job can be run after enabling this feature or changing the template, in order to apply the changes to the existing library.
```
